### ISSUE TITLE
minted and pygmentize

### ISSUE THREAD

**Issue by julmon** [dalibo/pandocker#27]
Hi,

I'm trying to use the latex package `minted` and I'm getting this error when I try to build a PDF file : 
```
! Package minted Error: You must have `pygmentize' installed to use this package
```
It would be really cool to have this package working. GH repo seems to be here https://github.com/gpoore/minted

Thank you !

**Comment by daamien** [comment]
https://packages.debian.org/stretch/texlive-latex-extra

**Comment by daamien** [comment]
should be fixed by #29
